.github/workflows/tests.yml

```
@@ -0,0 +1,48 @@
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 5
+      
+      - name: Run duplicate test
+        run: ./tests/duplicate
+
+  sorting-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)